src/coreclr/tools/aot/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/TransitionBlock.cs

```diff
@@ -387,8 +387,8 @@ public void ComputeReturnValueTreatment(CorElementType type, TypeHandle thRetTyp
                         }
                     }
 
-                    // Value types are returned using return buffer by default
-                    usesRetBuffer = true;
+                    // Value types are returned using return buffer by default except LoongArch64.
```

Comment:
This should match the logic in https://github.com/dotnet/runtime/blob/a9b18fc97a700f2012ced4042b470fefaf6af313/src/coreclr/vm/callingconvention.h#L1886-L1887 .

Reply:
Thanks

Reply:
Hi, @jkotas 
Are these `RETURN_HAS_RET_BUFFER` and `usesRetBuffer`  liking the X8_return_buffer for ARM64 ?

If these are not liking the ARM64's X8_return_buffer, there is no need to modify and I will revert the `usesRetBuffer = !IsLoongArch64`.
As the logic 
https://github.com/dotnet/runtime/blob/a9b18fc97a700f2012ced4042b470fefaf6af313/src/coreclr/vm/callingconvention.h#L1779-L1786
where it is only the TYPEDBYREF.

Reply:
Thanks
I had update it.